nodeEditor/src/diagram/graphFrame.ts

```diff
@@ -127,69 +134,125 @@ export class GraphFrame {
     }
 
     private _createFramePorts() {
+
         for (var node of this._nodes) {
             node.isVisible = false;
+        }
+
+        for (var port of this._exposedOutPorts) { // Output
+            if(port.node == null || port.node.enclosingFrameId != this.id) {

```

Comment:
nit - try as much as possible to use `===` instead of `==` . type safe and faster to execute

Reply:
Did you resolve it? I still see the ==

Reply:
yes you did 👍